src/test/java/com/wikia/webdriver/common/templates/fandom/AdsFandomTestTemplate.java

```diff
@@ -25,26 +25,10 @@ protected AdsFandomObject loadPage(String pageName, String pageType) {
         break;
     }
 
-    AdsFandomObject pageObject = new AdsFandomObject(driver, pageUrl);
-    getJquery();
-
-    return pageObject;
+    return new AdsFandomObject(driver, pageUrl);
   }
 
   protected AdsFandomObject loadPage(String pageName) {
     return loadPage(pageName, PAGE_TYPE_ARTICLE);
   }
-
-  private void getJquery() {
```

Comment:
You removed this function but inside project there is still a lot's of jQuery code, you can recognize it by this structure: $(  )
Lists of methods inside JavascriptActions.java:
```
com.wikia.webdriver.common.core.elemnt.JavascriptActions#click(java.lang.String)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#click(org.openqa.selenium.WebElement)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#focus(java.lang.String)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#focus(org.openqa.selenium.WebElement)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#mouseOver
com.wikia.webdriver.common.core.elemnt.JavascriptActions#scrollToElement(org.openqa.selenium.By)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#scrollToElement(org.openqa.selenium.WebElement)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#scrollToElement(org.openqa.selenium.WebElement, int)
com.wikia.webdriver.common.core.elemnt.JavascriptActions#scrollToElementInModal
```
I'm not sure if we really can remove everything :( is it only jQuery loader in our selenium-tests?